### PR TITLE
Switched to LiveSpeech Class for Hotword Detection

### PR DESCRIPTION
**Pull request by betterclever**
Fixes Issue #40 : Enable PocketSphinx LiveSpeech module Keyword spotting for Hotword Detection.

### CODE REVIEW

**Comment by betterclever** [comment]
@chiragw15 @dynamitechetan @saurabhjn76 @DravitLochan Please review

**Comment by betterclever** [comment]
@mariobehling Please merge.

File: app/main.py [hunk 1]
```diff
@@ -104,44 +103,15 @@ def start_speech_recognition():
...
 websocketThread.start()
 
-open_stream()
+liveSpeech = LiveSpeech(lm=False, keyphrase='susi', kws_threshold=1e-15)
 
-# TODO: Decide threshold by a training based system.
-# adjust threshold manually for now.
-sphinxRecognizer = SphinxRecognizer(threshold=1e-30)
+for phrase in liveSpeech:
+    print(phrase)
```
Comment (chashmeetsingh) [inline app/main.py]: This can be removed I guess?

Comment (betterclever) [inline app/main.py]: This does not affect much. This serves as additional logging output for the user.

**Pull request merged**